web/beacon-app/cypress/e2e/OrganizationDashboard/OrganizationDashboard.ts

```diff
@@ -0,0 +1,46 @@
+import { Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
+
+beforeEach(() => {
+  cy.loginWith({ email: 'mpie6802@gmail.com', password: 'elyseebleu1@gmail.com1A' });
+});
+
+Given('I login and navigate to the organization dashboard', () => {
+  cy.findByTestId('menu').click();
+  cy.findByTestId('settings').click();
+});
+
+Then('The displayed data are accurate', () => {
+  cy.findByTestId('menu').click();
+  cy.findByTestId('settings').click();
+
+  cy.findAllByRole('cell').contains("Elysee's Tenant");
+  cy.findAllByRole('cell').contains('Anthony Hill');
+  cy.findAllByRole('cell').contains('01GSZRN0P68J59EWK8JCXJ0H2R');
+  cy.findAllByRole('cell').contains('Thursday, February 23, 2023');
```

Comment:
If we log in with a different account, this will throw an error

Reply:
The login credentials are statics so they not going to change

Reply:
The same data will need to be provided if we decide tomorrow to add this automation under Github action, which won't always be possible since the organization ID is generated. It is recommended that you check to see if every cell contains a good pattern of data.

1- The string can be checked to see if it contains tenants or exist
2- You can check if the string is available and compare the ID's length or exist
3- Determine whether the string is a date format

Reply:
the ` cy.loginWith`  that you use is the authentication logic